blockchain/src/history/mmr_store.rs

```diff
@@ -232,6 +262,75 @@ pub fn remove_block_from_store(
     }
 }
 
+/// Copies the mmr entries of the previous block number
+/// This is used to initialize the tree for the given block number, using the previous tree as a baseline
+pub fn init_mmr_from_block_number(
```

Comment:
Not entirely sure why the MMR of a single block preceding the current one is the way to go when the MMR actually spans an entire epoch. A comment explaining this would be nice as it certainly threw me of and I cannot see mention as to why in this PR.

Reply:
I will update the comment